plumbum/cli/application.py

```diff
@@ -320,8 +317,39 @@ def _parse_args(self, argv):
                 else:
                     swfuncs[swinfo.func] = SwitchParseInfo(swname, (val,), index)
 
+        # Extracting arguments from environment variables
+        envindex = 0
+        for env, swinfo in self._switches_by_envar.items():
+            envindex-=1
+            envval = os.environ.get(env)
```

Comment:
since it's plumbum (after all), you should use ``local.env`` instead of ``os.environ``